ios/HackerNews/Feed/FeedScreen.swift

```diff
@@ -23,6 +23,7 @@ struct FeedScreen: View {
           .containerShape(.rect(cornerRadius: 24, style: .continuous))
 
         HStack(spacing: 16) {
+          Spacer()
```

Comment:
Can this be `Spacer(minLength: 0)`? Otherwise I think this will add default padding to the sides.

Reply:
Done :)